### PR TITLE
Switched to npm build scripts. Updated example readme

### PR DESCRIPTION
**Pull request by f1yn**
I removed the Makefile from `example01` and switched to inline npm scripts. `npm start` now builds and serves the project, `npm run build` will build the `dist.js` file, and `npm run watch` starts Rollup with the watcher.

### CODE REVIEW

File: example01/package.json [hunk 1]
```diff
@@ -1,6 +1,12 @@
 {
   "name": "example01",
   "private": true,
+  "scripts": {
+    "start": "npm run build",
+    "poststart": "./node_modules/serve/bin/serve.js .",
```
Comment (stasm) [inline example01/package.json]: It should be sufficient to just specify `serve .` here, as per https://docs.npmjs.com/misc/scripts#path.

File: example01/README.md [hunk 2]
```diff
@@ -4,4 +4,4 @@ A simple (and ulgy) Task Manager written using `innerself`.
 
 To install the dependencies and serve the example locally run:
 
-    $ make
+    $ npm start
```
Comment (stasm) [inline example01/README.md]: Can you add `$ npm install` as the first step here, to account for the `To install the dependencies…` part of the sentence?

**Review by stasm — APPROVED** [review]
Thanks for the PR! I just have two small comments.

**Comment by f1yn** [comment]
I've addressed those comments in the latest commit: https://github.com/stasm/innerself/pull/3/commits/a572f8d9c2f6b474bb27ce55c19eaa37f185e12c

**Comment by stasm** [comment]
Sweet, thanks!

**Pull request merged**